internal/pkg/flink/internal/store/store_test.go

```diff
@@ -542,7 +558,7 @@ func (s *StoreTestSuite) TestProccessHttpErrors() {
 
 	// expect
 	assert.NotNil(s.T(), err)
-	assert.Equal(s.T(), "Invalid syntax: you should provide a table for select", err.Error())
+	assert.Equal(s.T(), "Error: Invalid syntax: you should provide a table for select.", err.Error())
```

Comment:
Errors should not be capitalized and should not contain punctuation
```suggestion
	assert.Equal(s.T(), "Error: invalid syntax: you should provide a table for select", err.Error())
```

Reply:
Made all of them lowercase and removed the punctuation